### ISSUE TITLE
Tome of Peritia too hard to obtain

### ISSUE THREAD

**Issue by tjsd** [DarkPacks/SevTech-Ages#1392]
SEE https://github.com/DarkPacks/SevTech-Ages/issues/1392#issuecomment-381396867 FOR ISSUE

## Issue / Bug
Tome of Peritia is used to store XP and is available in Age 2 from blood magic.
Issue is that it requires Imbued Slates, those are Tier 3 and tier 3 altar require glowstone.

## Current Behavior
Can only be crafted if you have tier 3 blood altar

## Possible Solution
Change the slates to Tier 2 that is in stage 2.

scripts\staging\item_and_recipes\02_two\recipes.add.shaped.zs
Line: 570-576
	<bloodmagic:experience_tome> : [
		[
			[str, <ore:blockLapis>, str],
			[<bloodmagic:slate:1>, <betterwithmods:arcane_scroll:*>, <bloodmagic:slate:1>],
			[metals.gold.ingot, <bloodmagic:blood_orb:*>, metals.gold.ingot]
		]
	],

## Steps to Reproduce (for bugs)
1. Be in stage 2
2. Try to craft Tome of Peritia
3. Fail since you cant get tier III slates since that is not obtainable in stage 2.

## Your Environment
* Modpack Version: 3.0.4
* Java Version: 8 v144
* Launcher Used: MultiMC
* Server/LAN/Single Player: Server
* Resourcepack Enabled?: No
* Optifine Installed?: No

**Comment by tjsd** [comment]
I see there are other sigils also listed at the stage I am in now that require tier 3 blood magic. Are they just not hidden correctly? 

- Sigil of Magnetism (tier 3)
- Sigil of Haste (tier 4 or later)

**Comment by oOMitchOo** [comment]
Another possible solution would be to push these Blood Magic recipes into stage 3.

**Items which need a blood altar tier 3:**
1. Imbued Slate [tier 3 slate] (bloodmagic:slate:2) and therefore
2. Rune of Capacity [needs a tier 3 slate] (bloodmagic:blood_rune:6)
3. Displacement Rune [needs a tier 3 slate] (bloodmagic:blood_rune:5)
4. Tome of Peritia [needs a tier 3 slate] (bloodmagic:experience_tome)
5. Sigil of Magnetism [needs a tier 3 slate] (bloodmagic:sigil_magnetism)
6. Sigil of Holding [needs a tier 3 slate] (bloodmagic:sigil_holding)
7. Sigil of the Blood Lamp [needs a tier 3 slate] (bloodmagic:sigil_blood_light)
8. Magician Blood Orb [needs a tier 3 altar] (bloodmagic:blood_orb:0)
9. Weak Activation Crystal [needs a tier 3 altar] (bloodmagic:activation_crystal:0)
10. Elemental Inscription Tool: Water [needs a tier 3 altar] (bloodmagic:inscription_tool:1)

**Items which need a blood altar tier 4:**
1. Acceleration Rune [needs a tier 4 slate] (bloodmagic:blood_rune:9)
2. Sigil of Haste [needs a tier 4 slate] (bloodmagic:sigil_haste)
3. Rune of the Orb [needs at least a master blood orb] (bloodmagic:blood_rune:8)
4. Elemental Inscription Tool: Dusk [needs a tier 4 altar] (bloodmagic:inscription_tool:5)

**Items which are shown in JEI but don't have a recipe:**
1. Efficiency Rune (bloodmagic:blood_rune:2)
2. Sigil of the Whirlwind (bloodmagic:sigil_whirlwind)
3. Sigil of Winter's Breath (bloodmagic:sigil_frost)

**Comment by tjsd** [comment]
There are other stuff as well. It seems even the Tier III blood orb is not staged correctly.
This is in stage 2, but the blood orb used to create the Stone Brick Path is Tier 3
![image](https://user-images.githubusercontent.com/7138125/38330915-1eb195d0-3852-11e8-849b-f95637dcd66a.png)

**Comment by gekkobear** [comment]
7. Sigil of the Blood Lamp (the new recipe, candle, torch, 2 ash) only took a tier 2 slate in 3.0.4.  It's been very nice to have in Age 2 Twilight forest. :)

**Comment by Kaffee81** [comment]
currently (3.0.6) it also requires an BWM arcane scroll - protection, which according to the BWM book drop from slimes. with the drop rate of 0.001, it's just luck if you can even make the Tome of Peritia while playing this pack. how in the world am I supposed to kill thousands of slimes in age 2/3?

**Comment by dalsio** [comment]
Sea lanterns (harvested from or made using prismarine harvested from Ocean Monuments) can be used in place of Glowstone blocks for tier 3 Blood Altars. Arcane scrolls can be found as rewards in chests and other randomized loot; I found one much earlier than when I could ever use it and I don't think I've killed a single slime yet (I'm in age 4). The tome is also by far not the only way to collect and store XP, especially in age 3 and 4. It is true, however, that Arcane Scrolls are completely random and there's a chance you will never get one, but starting age 4 you gain access to various mob-farming methods that let you quickly kill and collect the drops/xp from many many mobs without much effort which makes getting a scroll almost an inevitability with some time/resource investment.

My current method of playing is to mostly ignore unnecessary advancements in each age and focus on progression, then once I've unlocked tools from the next age I go back and do all of the ones that were too much of a pain to do "the right way".
